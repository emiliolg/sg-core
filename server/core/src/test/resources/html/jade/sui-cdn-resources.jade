doctype html
html(sg-view="tekgenesis.service.html.Views")
    head
        title='Resources examples'
        link(rel="icon", href=sg.favicon())
        link(href= sg.css('theme.css') ).
        script(src= sg.js('lib.js') ).
    body
        img(src= sg.img('image.jpeg'))
        img(src= sg.img('/some/image.jpeg'))
        img(src= sg.resource('img/some/image.jpeg'))
        img(src= sg.sha('img/image.jpeg'))