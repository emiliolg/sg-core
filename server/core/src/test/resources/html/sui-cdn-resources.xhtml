<div sui-view="tekgenesis.service.html.Views">
    <img src="@Resource(img/image.jpeg)"/>
    <img src="@Sha(img/image.jpeg)"/>
</div>