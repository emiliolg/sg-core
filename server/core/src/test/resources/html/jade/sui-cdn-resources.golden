<!DOCTYPE html>
<html sg-view="tekgenesis.service.html.Views">
  <head>
    <title>Resources examples</title>
    <link rel="icon" href="/public/sg/img/favicon.png">
    <link href="http://cdnd.epi.ninja/css/theme.css">
    <script src="http://cdnd.epi.ninja/js/lib.js"></script>
  </head>
  <body><img src="http://cdnd.epi.ninja/sha/img/image.jpeg/c4bfa393f1"><img src="http://cdnd.epi.ninja/img/some/image.jpeg"><img src="http://cdnd.epi.ninja/img/some/image.jpeg"><img src="http://cdnd.epi.ninja/public/img/image.jpeg"></body>
</html>